UKHomeOffice/nmsw-ui
# NMSW-1377 Update UPT result status banners as there is no green & valid are now checks

# Ticket



----

## AC

Given I have uploaded a valid FAL6
When all passengers have valid permission to travel
Then I should see the blue banner and all passengers should be included in the 'check documents' count

Given I have uploaded a FAL6
When some passengers are valid permission to travel
Then I should see the blue banner and those passengers should be included in the 'check documents' count


----

## To test

- upload a FAL6 where all passengers have valid permission
- you should see the blue banner with a count totalling all those passengers for 'check'
- upload a FAL6 where some passengers have valid permission and some are do not board
- you should see the blue banner with total passengers - do not board passengers = check

----

## Notes
